test/offline/storage_integration.js

```diff
@@ -948,30 +948,6 @@ describe('Storage', () => {
           .toBeRejectedWith(expected);
     });
 
-    it('throws an error if DRM sessions are not ready', async () => {
```

Comment:
In addition to removing the old test, I believe we need a new test to verify the new behavior.  This needs to be tested on all supported platforms, and an automated test is probably the best way.

Reply:
I have been watching and there is no test for persistent licenses. Right now I would not know how to do the test :( I think it is better to launch an issue to add tests of persistent licenses.

Reply:
That's fair.  I filed #2168 to track that.